source/messaging/organizing-channels.rst

```diff
@@ -12,9 +10,7 @@ This document shares best practices from various Mattermost deployments includin
 The importance of channel names
 -------------------------------
 
-Channels organize communication in Mattermost.
-
-When naming or renaming channels, consider the following:
+Channels organize communication in Mattermost Channels. When naming or renaming channels, consider the following recommendations:
```

Comment:
You can name and rename your conversations in Mattermost Channels. When you're choosing names, consider the following recommendations: